### ISSUE TITLE
Great work! would you mind providing data synthetic code?

### ISSUE THREAD

**Issue by sjtuytc** [cvlab-epfl/segmentation-driven-pose#3]
Hello, we're trying to reimplement training code but we need to make a synthetic dataset first.
Would you mind share part of the script?
If not, can you give us more details? Do you try to prevent objects from overlapping?

**Comment by YinlinHu** [comment]
For YCB-Video dataset, the synthetic part is very simple, just random background is OK. For Occluded-LINEMOD dataset, we also use a very simple scheme: for each image sample, just generate a random combination of different objects. During this procedure, the positions, rotations, sizes are random, so, it will have overlapping. While as the generation procedure is known, the ground truth mask can be easily obtained.
